notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<p>can you please add a sentence about the graph? what we are seeing here? what's on the x-axis and whats on the y-axis?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Sure</p>